sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.Request.cs

```diff
@@ -13,36 +14,10 @@ namespace Azure.Core.Pipeline
     /// </summary>
     public partial class HttpClientTransport : HttpPipelineTransport, IDisposable
     {
-        private sealed class HttpClientTransportRequest : Request
+        private sealed class HttpClientTransportRequest : HttpPipelineRequest
```

Comment:
This is the crux of this.  The transport implementations of request and response inherit from the S.SM.R implementations to reuse that functionality.